roles/docker_storage_to_overlay2/vars/main.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
These should probably move to `defaults/main.yml` so they can be overridden if needed.  Any static variables live in `vars/main.yml`.

Reply:
Hmm, I didn't intend for any of these to be overridden.  Is there a use case when you would want to?

I could maybe see `docker_storage_to_overlay2_optional_service_names`, but I think I'd rather provide a way to amend the list, not replace it.

Reply:
My preference is to put things in defaults/main.yml unless there's a specific reason not to.